R/net.utils.R

```diff
@@ -717,6 +717,10 @@ get_network_term_attr <- function(nw) {
 #' idgroup(nw, ids = c(3, 6))
 #'
 idgroup <- function(nw, ids) {
+
+  if (class(nw) == "list") {
+    nw <- nw[[1]]
```

Comment:
why is this necessary? the nw argument is defined as a network object. So it would be expected that if it were embedded in a sublist, one would pass in idgroup(nw[[1]], ...)

Reply:
@smjenness this was put in to differentiate between cases used outside of EpiModel and within the EpiModel workflow. This was a change mainly to accommodate the examples provided with this function.

Reply:
Yup, revert this

Reply:
Reverted.